automation/features/step-definitions/staking.steps.js

```diff
@@ -1,132 +1,179 @@
 })
 
-When(/^I click on use maximum button$/,()=>{
-    browser.getByTestId('callout').waitForDisplayed({timeout:30000,reverse:false,timeoutMsg: "callout was not found"})
-    if (stakingPage.associateMoreTokensBtn.isDisplayed()){
-    stakingPage.associateMoreTokensBtn.click()
-  } else stakingPage.associateTokensBtn.click()
-  $('[data-testid="associate-radio-wallet"]').click({force:true})
-  currentAmountOfTokensInWallet = browser.getByTestId('eth-wallet-balance').getText()
-  browser.getByTestId("token-amount-use-maximum").click()
+When(/^I click on use maximum button$/, () => {
+    browser.getByTestId('callout').waitForDisplayed({ timeout: 30000, reverse: false, timeoutMsg: "callout was not found" })
+    if (stakingPage.associateMoreTokensBtn.isDisplayed()) {
+        stakingPage.associateMoreTokensBtn.click()
+    } else stakingPage.associateTokensBtn.click()
+    $('[data-testid="associate-radio-wallet"]').click({ force: true })
+    currentAmountOfTokensInWallet = browser.getByTestId('eth-wallet-balance').getText()
+    browser.getByTestId("token-amount-use-maximum").click()
 })
 
-When(/^I can see the maximum amount of tokens in my wallet are in the token input box$/,()=>{
-  const noZeroes = currentAmountOfTokensInWallet.toString();
-  const noZeroesFloat = parseFloat(noZeroes.replace(",",""))
-  expect(browser.getByTestId('token-amount-input').getValue()).toBe(String(noZeroesFloat))
+When(/^I can see the maximum amount of tokens in my wallet are in the token input box$/, () => {
+    const noZeroes = currentAmountOfTokensInWallet.toString();
+    const noZeroesFloat = parseFloat(noZeroes.replace(",", ""))
+    expect(browser.getByTestId('token-amount-input').getValue()).toBe(String(noZeroesFloat))
 })
 
-Then(/^I can see the validator node list$/,()=>{
-expect(browser.getByTestId('validator-node-list')).toBeDisplayed()
-  const nodeList = $(".node-list").$$('[data-testid="node-list-item"]');
-  expect(nodeList.length).toBeGreaterThan(0);
+Then(/^I can see the validator node list$/, () => {
+    expect(browser.getByTestId('validator-node-list')).toBeDisplayed()
+    const nodeList = $(".node-list").$$('[data-testid="node-list-item"]');
+    expect(nodeList.length).toBeGreaterThan(0);
 })
 
-Then(/^the epoch countdown timer is counting down$/,()=>{
-const currentCountdownTimerText = browser.getByTestId('current-epoch-ends-in').getText()
-browser.pause(2100) // let some time pass 
-expect(browser.getByTestId('current-epoch-ends-in').getText()).not.toEqual(currentCountdownTimerText)
+Then(/^the epoch countdown timer is counting down$/, () => {
+    const currentCountdownTimerText = browser.getByTestId('current-epoch-ends-in').getText()
+    browser.pause(2100) // let some time pass 
+    expect(browser.getByTestId('current-epoch-ends-in').getText()).not.toEqual(currentCountdownTimerText)
 })
 
-Then(/^I pause some seconds$/,()=>{
-browser.pause(600000)
+Then(/^I pause some seconds$/, () => {
+    browser.pause(600000)
```

Comment:
Unused step?

Reply:
it is used for debugging but i have removed it here